README.md

```diff
@@ -1,36 +1,58 @@
-# CI/CD Bootcamp
+# CIC Bootcamp
 ## Introduction
-Welcome to the CI/CD/IaC Bootcamp. The Bootcamp is a set of materials designed to teach you how to write tests for and automate the release of the things you build.
-
-## What's in the Bootcamp
-The course is being actively built out so be sure to pull regularly to get new exercises as they appear.
-### Infrastructure as code
-#### Ansible
-#### Testing
-## Getting Started
+This course is intended to teach good practices around Infrastructure-as-Code development. This course contains a set of materials which are designed to teach you how to write tests for and automate the release of the things you build.
+
+To undertake this course you will need a computer which you have full control over (i.e not your employer's desktop build) - It will need to be on a network with unrestricted access to the Internet (not behind locked-down corporate firewalls) as you will need to download and install a few components before you can begin.
+
```

Comment:
Is it worth adding a minimum spec paragraph here... pointing out that the course uses both virtualisation and containerisation and therefore has reasonably high memory requirements. Also require Intel VT or AMD-V virtualisation extensions turned on in the bios to support Virtualbox.

Reply:
Worth mentioning if you think that it's likely to happen and the help given by VirtualBox and the likes is not going to be very helpful. Could you put together a paragraph?